### ISSUE TITLE
auto_scrape_names false + overwrite_files true

### ISSUE THREAD

**Issue by WellSomething** [DIGITALCRIMINAL/OnlyFans#1159]
Hello,

I'm trying to figure out if these 2 variables are working as it should. First I would've thought that if auto_scrape_names was set to false, once you got the list of models and you chose one, then the script would one work with that one model. Instead the times I've used that, it downloads content from that model and also from a lot of other models (not sure if all).

So under the same idea, if I set overwrite_files to true, it doesn't redownload the files for that specific model, but it seems to redownload just about everything. Like last night it was downloading almost a terabyte of content when I just wanted to redownload 1 model. I had to stop the script. On 6.4.4 (I'm using 7.0.1). the same thing happened, but not to this magnitude... so not sure why in 6.4.4 it redownloaded a lot of extra files, yet not to the extent of almost a terabyte of content.

Is this how it's supposed to work?

Thanks

**Comment by artyom2035** [comment]
What's set under "jobs"?  If `auto_scrape_names` is `false` and `scrape_paid_content` is `true` other names will show because it'll scrape for paid content (even expired subs). If paid is on, try turning it off to see if that changes anything.

Now, I've never gotten `overwrite_files` to work as I think it was intended. If `auto_scrape_names` is on when I try to to use it to re-download a single, mistakenly deleted file it redownloads everything from all subs.  I thought  it  was due to my NAS' Linux kernel not tracking 'create' times the way NTFS does.  I leave it off because it'll redownload everything on every pass.  To re-download a single model - I set `auto_scrape_names` to `false`, `auto_choice` to `a`,  'auto_scrape_apis` to `true`, `scrape_names` to `true`,  `scrape_paid_content` to `false`
